### ISSUE TITLE
Abnormal pulmonary alveolar system morphology

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6051]
**Preferred term label:**
Abnormal pulmonary alveolar system morphology

**Synonyms**

**Definition (free text, please give PubMed ID)**
A structural abnormality of the pulmonary acinus, alveolar parenchyma, or alveoli.

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0002088

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
as discussed today @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
added term
